src/main.ts

```diff
@@ -0,0 +1,49 @@
+import * as core from '@actions/core'
+import {getS3Object} from './s3'
+import {FrozenBranches} from './fileHandler'
+
+import {
+  branchName,
+  bucketName,
+  deploy_environment,
+  targetBranch
+} from './config'
+
+const event_type = process.env.GITHUB_EVENT_NAME
+core.info('event_type: ' + event_type)
+export const eventType = core.getInput('EVENT_TYPE')
+
+async function run(): Promise<void> {
+  const s3FileKey = `assist/${deploy_environment}.json`
+  let baseBranch = branchName
+  if (event_type !== 'push') baseBranch = targetBranch
+  core.info(` The base branch is ${baseBranch}`)
+
+  let frozenBranches
+  let isBranchPresent
+
+  try {
+    const frozenBranchData = await getS3Object(bucketName, s3FileKey)
+    if (!frozenBranchData) {
+      core.info(
```

Comment:
Remove the log and simplify the if condition (remove else)

Reply:
Done.